2020q1/ena.md

```diff
@@ -0,0 +1,31 @@
+## ENA FreeBSD Driver Update ##
+
+Link: [ENA README](https://github.com/amzn/amzn-drivers/blob/master/kernel/fbsd/ena/README)
+
+Contact: Michal Krawczyk <mk@semihalf.com>  
+Contact: Maciej Bielski <mba@semihalf.com>  
+Contact: Marcin Wojtas <mw@semihalf.com>  
+
+ENA (Elastic Network Adapter) is the smart NIC available in the
+virtualized environment of Amazon Web Services (AWS). The ENA
+driver supports multiple transmit and receive queues and can handle
+up to 100 Gb/s of network traffic, depending on the instance type
+on which it is used.
+
+Completed since the last update:
+  * Upstream of the driver v2.1.1 version, introducing:
```

Comment:
s/driver v2.1.1 version/driver to v2.1.1/

Reply:
Done